src/server/game/Entities/Player/Player.cpp

```diff
@@ -24836,7 +24836,7 @@ void Player::StoreLootItem(uint8 lootSlot, Loot* loot)
 
     if (!item || item->is_looted)
     {
-        SendEquipError(EQUIP_ERR_ALREADY_LOOTED, nullptr, nullptr);
+        //SendEquipError(EQUIP_ERR_ALREADY_LOOTED, nullptr, nullptr);
```

Comment:
What does this do?

Reply:
Let me see the best way to describe this.
You may see an “error” message saying you've already looted this creature when dealing with looting a creature with in the radius of 30, every time you loot multi corpses with in radius of 30. This would eliminate that message always happening.
At this time I guess u can say its a place holder for when i add the optional confs, just not having the time due to a last minute irl issue a few days ago when i got with aokromes initially about this pr.